plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/PointStroke.tsx

```diff
@@ -1,9 +1,186 @@
-import { BaseFieldProps } from "../types";
+import AddButton from "@web/extensions/sidebar/core/components/content/common/AddButton";
+import { styled } from "@web/theme";
+import { useCallback, useState } from "react";
+
+import { BaseFieldProps, Cond, Expression, Fields } from "../types";
+
+import { array_move } from "./common/common";
+import ConditionField from "./common/ConditionField";
+import Field from "./common/Field";
+import ItemControls from "./common/ItemControls";
 
 const PointStroke: React.FC<BaseFieldProps<"pointStroke">> = ({ value, editMode, onUpdate }) => {
   // remember to update the BaseFieldProps type!
-  console.log(value, editMode, onUpdate);
-  return <div>PointColor</div>;
+  const [conditions, updateConditions] = useState(value.conditions);
```

Comment:
The `// remember to update the BaseFieldProps type!` comment was for you :P If you don't need it anymore please remove it

Reply:
Done!